fh_immuta_utils/policy.py

```diff
@@ -226,21 +227,26 @@ def _read_subscription_configs(self, config_root: str) -> None:
         for subscription_policy_file in glob.glob(
             os.path.join(config_root, "policies/subscription", "*.yml")
         ):
-            logging.debug("Reading subscription policy file: %s", subscription_policy_file)
+            logging.debug(
+                "Reading subscription policy file: %s", subscription_policy_file
+            )
             with open(subscription_policy_file) as handle:
                 contents = yaml.safe_load(handle)
                 self.subscription_policy_config = {
                     **self.subscription_policy_config,
                     **contents.get("SUBSCRIPTION_POLICIES", {}),
                 }
 
-
-
     def read_configs(self, config_root: str) -> None:
         self._read_data_configs(config_root=config_root)
         self._read_subscription_configs(config_root=config_root)
 
 
+# https://docs.python.org/3/distutils/apiref.html#distutils.util.strtobool
+def str2bool(v: str) -> bool:
+    return bool(strtobool(v))
```

Comment:
interesting that `strtobool` doesn't return `True` or `False`, but `1` and `0`. looks like someone tried to submit a fix for it but it was shot down: https://bugs.python.org/issue27721